.changeset/strong-hounds-work.md

```diff
@@ -0,0 +1,5 @@
+---
+"@turnkey/http": minor
+---
+
+Update to v2024.2.1 API: add InitImportWallet and ImportWallet activities
```

Comment:
We should add the new delete endpoints to the changelog: delete users, delete private key tags, delete user tags: all possible now!

Reply:
Oh! And we have a new `list_suborgs` endpoints to list sub organizations by email, public key, or credential ID!

Reply:
oh awesome, i didn't realize when i made the changeset! i'll add those now